cs251tk/toolkit/args.py

```diff
@@ -165,4 +168,62 @@ def process_args():
     assignments = get_assignments_from_args(**args)
     stogit = compute_stogit_url(**args, _now=datetime.date.today())
 
+    print_args(args)
+    print_students(students)
+    print_assignments(assignments)
+    debug("stogit URL: " + stogit)
+
     return args, students, assignments, stogit
+
+
+def print_args(args):
+    debug("Command Line Arguments:")
+    debug(args)
+    debug("version:                 " + str(args['version']))
+    debug("debug:                   " + str(args['debug']))
+    debug("skip_update_check:       " + str(args['skip_update_check']))
+    debug("course:                  " + str(args['course']))
+    debug("students:                " + str(args['students']))
+    debug("sections:                " + str(args['sections']))
+    debug("all_sections:            " + str(args['all_sections']))
+    debug("quiet:                   " + str(args['quiet']))
+    debug("no_progress:             " + str(args['no_progress']))
+    debug("workers:                 " + str(args['workers']))
+    debug("sort_by:                 " + str(args['sort_by']))
+    debug("highlight_partials:      " + str(args['highlight_partials']))
+    debug("clean:                   " + str(args['clean']))
+    debug("no_update:               " + str(args['no_update']))
+    debug("stogit:                  " + str(args['stogit']))
+    debug("date:                    " + str(args['date']))
+    debug("no_check:                " + str(args['no_check']))
+    debug("to_record:               " + str(args['to_record']))
+    debug("gist:                    " + str(args['gist']))
+    debug("interact:                " + str(args['interact']))
```

Comment:
Sorry, GitHub won't do multi-line suggestions yet – what I meant to ask was, instead of printing each of these arguments individually, could we replace all of these lines with the one `debug(args)` line?

Reply:
I ended up creating a loop that formatted them the same way